sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,19 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
```

Comment:
it's failing when trying to pass a string to the fit method of a hashingvectorizer and not getting back the valueerror, probably because you haven't fit it here (https://github.com/scikit-learn/scikit-learn/pull/7507/files#diff-904f5f849e36c648b8b05de140d7fad8R452). That being said, you maybe don't need to add anything, since `HashingVectorizer` is stateless and fit does nothing anyway.